hawtio-system/src/main/java/io/hawt/web/auth/AuthenticationConfiguration.java

```diff
@@ -205,7 +249,7 @@ private static List<AuthenticationContainerDiscovery> getDiscoveries(String auth
                 AuthenticationContainerDiscovery discovery = clazz.getDeclaredConstructor().newInstance();
                 discoveries.add(discovery);
             } catch (Exception e) {
-                LOG.warn("Couldn't instantiate discovery " + discoveryClass, e);
+                LOG.warn("Couldn't instantiate discovery {}", discoveryClass, e);
```

Comment:
It may look like this, but implementations do some clever rearrangement to detect throwable among params:
* https://github.com/qos-ch/logback/blob/v_1.5.6/logback-classic/src/main/java/ch/qos/logback/classic/spi/LoggingEvent.java#L141
* https://github.com/apache/logging-log4j2/blob/rel/2.23.1/log4j-api/src/main/java/org/apache/logging/log4j/message/ParameterizedMessage.java#L158-L164